### ISSUE TITLE
Communication

### ISSUE THREAD

**Issue by koaning** [RasaHQ/whatlies#121]
I am wondering if it makes sense to invest in a communication channel. Sometimes there are things I prefer handling on a github issue but other times something like instant messaging might be better. 

For example. @mkaze I pushed a new version live today with a nifty feature: https://rasahq.github.io/whatlies/tutorial/scikit-learn/ 

Our language backends now also support Scikit-Learn. That means that you can easily hook up a logistic regression after pretrained word embeddings to check if they fit your use-case. 

Would something like gitter make sense for this sort of thing? I can also set up a slack channel for people who have invested a lot in this project.

**Comment by mkaze** [comment]
> For example. @mkaze I pushed a new version live today with a nifty feature: https://rasahq.github.io/whatlies/tutorial/scikit-learn/

Cool, that's a nice feature!

> I am wondering if it makes sense to invest in a communication channel. Sometimes there are things I prefer handling on a github issue but other times something like instant messaging might be better.

Yeah, maybe! For example, things like announcement of new features/changes/releases (although, the release section of Github is usually used for that) or getting feedback from users or discussing changes in a more relaxed way, probably could be done in a much better way in something like gitter or a slack channel. Although, it seems the user base is still small as I don't see a lot of issues raised by them (this might be partly due to the applicability of the project which is more geared towards exploration and gaining an understanding rather than building something).

**Comment by koaning** [comment]
There's two options in my mind for now. 

1. I can start something on gitter. The upside: it's open. The downside: because it is open you can quickly loose structure. 
2. I have a slack room that I could invite people in but this will put a limit on the folks who can join. 

@mkaze let me know if you've got a preference.

**Comment by mkaze** [comment]
@koaning Yeah, either one has its own cons/pros. I don't have a specific preference there; whatever works best for you.

**Comment by koaning** [comment]
I can invite you to the Rasa contributor slack. We can add a room there so people who are interested can talk/hang out. 

What email can I use for the invite? After adding you to the channel we can remove your email from this thread.

**Comment by mkaze** [comment]
Thanks! You can use the email in my SO profile here: https://stackoverflow.com/users/2099607/today?tab=profile

**Comment by mkaze** [comment]
@koaning BTW, did you do any announcement or advertise this project recently?! I am asking this because the number of "stargazers" of this repo is strangely rising today?!! :thinking: Although, whatever the reason, it's a good thing that people are becoming aware of its existence! :+1:

**Comment by koaning** [comment]
Haha, yes I did announce the scikit-learn feature today. From my end; yes -> it's a popular announcement.

**Comment by mkaze** [comment]
Ah, I see! I think at least 10 user starred this repository today! That's good..... but I also have a fixed feeling of fear and happiness that they might look at this as a way of using embeddings in scikit-learn for building NLP pipelines (which is not the focus this project). 
Although, considering the lack of extensive support for NLP or CV data preparation or transformation in scikit-learn, that's not a surprise to me; it might be even an opportunity for interested developers to write some (unofficial) extension for scikit-learn in these domains.

**Comment by koaning** [comment]
I might argue that good integration with scikit learn will make it easier to predict a label, which in turn can be used to color dots in a scatter chart. In that sense I would still say it's very much aligned with the idea of trying to understand 'whatlies' in word embeddings. 

But yeah, if people want us to support the pickling of all of these backends then I'm fine with saying "no". That'd be the job for another package.

**Comment by koaning** [comment]
It's funny you mention it though. I also maintain [scikit-lego](https://github.com/koaning/scikit-lego) and this project is now recognized as a [recognized project](https://scikit-learn.org/dev/related_projects.html).

**Comment by mkaze** [comment]
@koaning Yeah, I have already seen that before. Nice work!
